services/src/api/handlers/datasets.rs

```diff
@@ -378,6 +378,8 @@ pub async fn create_upload_dataset<C: ApplicationContext>(
     let db = app_ctx.session_context(session).db();
     let upload = db.load_upload(upload_id).await.context(UploadNotFound)?;
 
```

Comment:
comment unrelated to this line:

I think this approach in general is fine. To make Barbara even more happy I think it would be nice if one could navigate back in the UI by means of the back button next to the breadcrumbs instead of the back button on the top right

Reply:
i guess thats a new task?